portfolio/src/main/webapp/script.js

```diff
@@ -42,3 +42,24 @@ function showSlides(newIndex) {
   slides[slideIndex].style.display = "block";
   dots[slideIndex].className += " active";
 }
+
+/**
+ * Displays list of comments returned by server.
```

Comment:
Nit. Try to write full sentences in comments: "Displays the list of comments returned by the server." This is easier to read.